schema/vrs-source.yaml

```diff
@@ -411,18 +443,27 @@ definitions:
 
   CytobandInterval:
     description: >-
-      A contiguous region specified by chromosomal bands features.
+      A contiguous region specified by cytobands. The region includes
```

Comment:
For consistency this should be stated similarly to SequenceLocation "the span on a chromosome bounded by ctyoband endpoints" or something to that affect. It is not a region or a cytoband itself. It is an "interval" that is strictly used by the ChromosomeLocation to define a location on a chromosome based on cytoband nomenclature.

Reply:
Added #361 to address later.